src/components/Table/components/TableCell/TableCell.tsx

```diff
@@ -69,7 +71,7 @@ const TableCell: React.FC<Props> = React.memo(
             fontWeight: theme.typography.weights.bold,
             fontSize: theme.typography.fontSizes['14'],
           },
-          component === 'th' && isSortable && { ...parentStyles()(theme) },
+          component === 'th' && isSortable && { ...parentStyles(isActive)(theme) },
```

Comment:
something i skipped is this. Can you use object here. Is better for extendability of the function in params. 

`parentStyles({ isActive })`